README.md

```diff
@@ -668,6 +676,10 @@ If you're using the `X-SourceMap` header instead, you can just omit `sourceMap.u
   compressor from discarding unused function arguments.  You need this
   for code which relies on `Function.length`.
 
+- `keep_classnames` (default: `false`) -- Pass `true` to prevent the
+  compressor from discarding class names.  See also: the `keep_classnames`
+  [mangle option](#mangle).
+  
```

Comment:
Minor nit - shouldn't this be above `keep_fargs` alphabetically?

Reply:
And this is when I notice ``(default: `true`) -- default `true`.`` seems pretty redundant :ghost:

Reply:
keep_fargs - will fix.

Where is `(default: `true`) -- default `true`?

Reply:
> Where is ``(default:`true`) -- default `true`.``?

It is in `keep_fargs` - I'll fix it on `master`.

Reply:
default... default - I see it now.

Reply:
> It is in keep_fargs - I'll fix it on master.

That's worthy of both a 👍 and a :taco:.